### PR TITLE
Feature/407 finger print login

### PR DESCRIPTION
**Pull request by HaseebUllahAbbasi**
finger print setup after the  Login and Pin

### CODE REVIEW

**Comment by theblockstalk** [comment]
I changed the base branch from `master` to `development`

**Comment by theblockstalk** [comment]
I'm confused, why is this PR linkedin to issue related to fingerprint, but the PR is fixes for README.md and PIN screen?

File: src/containers/LoginPinScreenContainer.tsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ export default function LoginPinScreenContainer({
         setDisabled(pin.length < 5);
         setErrorMessage('');
     }
-
+    function onSkip() {
+        navigation.navigate('UserHome');
```
Comment (theblockstalk) [inline src/containers/LoginPinScreenContainer.tsx:34]: on skip should take to the Fingerprint setup screen

File: src/containers/LoginPinScreenContainer.tsx [hunk 2]
```diff
@@ -18,6 +18,10 @@ export default function LoginPinScreenContainer({
 }) {
     const [disabled, setDisabled] = useState(true);
     const [pin, setPin] = useState('');
+    /***
+     * The Correct pin set 11111, it can be fetched and set of user from storage
+     *
+     * */
     const [confirmPin, setConfirmPin] = useState('11111');
```
Comment (theblockstalk) [inline src/containers/LoginPinScreenContainer.tsx:25]: we should not have a default confirmPin, the user needs to confirm the PIN

File: src/containers/LoginPinScreenContainer.tsx [hunk 3]
```diff
@@ -30,13 +34,26 @@ export default function LoginPinScreenContainer({
         setDisabled(pin.length < 5);
         setErrorMessage('');
     }
-
+    /***
+     *  Navigated to Create Finger Print Screen on Skip
```
Comment (theblockstalk) [inline src/containers/LoginPinScreenContainer.tsx:38]: if comment is only one line. done use multi-line comment
```typescript
// Navigated to Create Finger Print Screen on Skip
```

Comment (theblockstalk) [reply src/containers/LoginPinScreenContainer.tsx:38]: also fix other lines

Comment (HaseebUllahAbbasi) [reply src/containers/LoginPinScreenContainer.tsx:38]: sure , Thanks @theblockstalk

**Review by theblockstalk — COMMENTED** [review]
seems like https://github.com/Tonomy-Foundation/Tonomy-ID/pull/425 will make all these changes

Can you please ensure all PRs are independent.

**Comment by theblockstalk** [comment]
I think this can be merged now. as any other changes will be done in #425

**Pull request merged**